scripts/src/utils/utils.ts

```diff
@@ -242,3 +242,18 @@ export function cached(cache: RedisAsyncClient, keygen: (...args: any[]) => stri
 		return descriptor;
 	};
 }
+
+export type Memo = {
+	version: string;
+	appId: string;
+	orderId: string;
+};
+
+export function parseMemo(memo: string): Memo{
+	const parts = memo.split("-");
```

Comment:
maybe add a comment with an example memo, it's usually easier to comprehend a visual. Not super important though.

Reply:
done